### PR TITLE
fix exception in launch test on localhost: This exception might occur…

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

**Comment by flopezag** [comment]
Check the Travis execution, all tests failed...

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/EnvironmentInstanceManagerImplTest.java [hunk 1]
```diff
@@ -32,14 +32,12 @@
 import static org.mockito.Mockito.when;
 
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
-import org.junit.Before;
+import com.telefonica.euro_iaas.paasmanager.exception.*;
```
Comment (jframos) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/EnvironmentInstanceManagerImplTest.java]: Do you use all exceptions of that Package?

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/EnvironmentInstanceManagerImplTest.java [hunk 2]
```diff
@@ -280,8 +293,27 @@ public void testCreateEnvironmentInstance() throws InfrastructureException, Enti
...
+        EnvironmentInstanceManagerImpl environmentInstanceManager = new EnvironmentInstanceManagerImpl();
+        ClaudiaData claudiaData = new ClaudiaData("org", "vdc", "service");
+        EnvironmentInstance environmentInstance = new EnvironmentInstance();
+        environmentInstance.setName("name");
+        Environment environment = new Environment();
+        environment.setName("environmentName");
+        environmentInstance.setEnvironment(environment);
+        environmentInstance.setBlueprintName("blueprint name");
+        environmentInstance.setTierInstances(new ArrayList<TierInstance>());
+        EnvironmentInstanceDao environmentInstanceDao= mock(EnvironmentInstanceDao.class);
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/EnvironmentInstanceManagerImplTest.java]: white space before =

**Comment by flopezag** [comment]
LGTM

**Pull request merged**